test_utils/fake_catalog_api.py

```diff
@@ -5,8 +5,12 @@
 
 from __future__ import absolute_import, unicode_literals
 
+import uuid
+
 from six.moves import reduce as six_reduce
 
+FAKE_UUIDS = [str(uuid.uuid4()) for i in range(10)]  # pylint: disable=no-member
```

Comment:
Well I tried to make it this way in case someone later needs to utilize many UUIDs where overlap isn't possible, but come to think of it I doubt someone would straight up need 10 different ones so I'll reduce it to 5.